src/raft.rs

```diff
@@ -955,6 +955,27 @@ impl<T: Storage> Raft<T> {
                 // will not create disruptive term increases
                 let to_send = new_message(m.get_from(), MessageType::MsgAppendResponse, None);
                 self.send(to_send);
+            } else if m.get_msg_type() == MessageType::MsgRequestPreVote {
+                // Before pre_vote enable, there may have candidate with higher term,
```

Comment:
The comment is confusing. I thought `candidate` here refers to the one who sends prevote request. Turns out it refers to the receiver. Could you make it better?